### PR TITLE
Support remote pagination

### PR DESCRIPTION
**Pull request by jshow**
Closes https://github.com/activeadmin/activeadmin/issues/5669.

Replaces https://github.com/activeadmin/activeadmin/pull/5667.

### CODE REVIEW

File: docs/8-custom-actions.md [hunk 1]
```diff
@@ -30,6 +30,44 @@ end
 This collection action will generate a route at `/admin/posts/import_csv`
 pointing to the `Admin::PostsController#import_csv` controller action.
 
+To paginate a table using ajax within a page, the following aproach works with Kaminari
+
+```ruby
+ActiveAdmin.register Post do
+  collection_action :member_posts, method: :get do
+    render 'admin/members/member_posts', context: self
+  end
```
Comment (deivid-rodriguez) [inline docs/8-custom-actions.md]: Shouldn't this be added to the `Member` admin page?

File: docs/8-custom-actions.md [hunk 2]
```diff
@@ -30,6 +30,44 @@ end
...
+
+  span id: 'member_posts' do
+    render partial: 'admin/members/member_posts'
+  end
```
Comment (deivid-rodriguez) [inline docs/8-custom-actions.md]: I wanted to try this, but I'm not sure in which context this is supposed to be rendered. Is it in the show page? :thinking:

File: docs/8-custom-actions.md [hunk 3]
```diff
@@ -30,6 +30,44 @@ end
...
+# app/views/admin/members/_member_posts.html.arb
+member = Member.find(params[:id])
+
+panel "#{member.name} Posts" do
+  columns do
+    paginated_collection(member.posts.page(params[:page]).per(20), remote: true, params: {controller: 'admin/members', action: 'member_posts'}, param_name: :page, download_links: false) do
+      table_for(collection, :class => 'index_table') do
+        column 'ID', :id
+
+        column 'Title', :title do |story|
```
Comment (deivid-rodriguez) [inline docs/8-custom-actions.md]: This block variable should be `post` I guess?

File: docs/8-custom-actions.md (hunk 2 above)
Comment (jshow) [reply docs/8-custom-actions.md]: the above partial (for which an example is below) is an #index - render an index of all `posts` for a `member`

**Comment by jshow** [comment]
thanks for your comments @deivid-rodriguez

File: docs/8-custom-actions.md (hunk 2 above)
Comment (deivid-rodriguez) [reply docs/8-custom-actions.md]: But I then you need to wrap the `span` helper with an `index do ... end` block, right?

**Pull request closed without merging**